flow/embedded_views.h

```diff
@@ -250,6 +250,14 @@ class ExternalViewEmbedder {
 
   virtual bool SubmitFrame(GrContext* context);
 
+  // Caller should make sure to call this after |SubmitFrame|.
+  // Embedder that implements this method to do additional tasks after
```

Comment:
`This method provides the embedder a way to do additional tasks after |SubmitFrame|`.

Reply:
done